MAUI/Picker/date-picker.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: DatePicker with .NET MAUI Picker Control | Syncfusion
+description: Learn here all about date picker with Syncfusion .NET MAUI Picker (SfPicker) control.
+platform: maui
```

Comment:
Confirm DatePicker, DateTimepicker and time picker need to mention in picker control?

Reply:
As per conversation above mentioned topic removed